include/deal.II/grid/grid_generator.h

```diff
@@ -2020,14 +2020,16 @@ namespace GridGenerator
    * (quadrilaterals, hexahedra) to a triangulation only consisting of
    * simplices (triangles, tetrahedra).
    *
-   * As an example, the following image shows how a set of three hexahedra
+   * As an example, the following image shows how a set of four hexahedra
    * meshing one eighths of a sphere are subdivided into tetrahedra, and how
```

Comment:
Not your patch but since we're here:
```suggestion
   * meshing one eighth of a sphere are subdivided into tetrahedra, and how
```

Reply:
Good point. So fixed!